new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java

```diff
@@ -30,10 +33,13 @@
 import static oracle.weblogic.kubernetes.assertions.impl.Kubernetes.isPodReady;
 import static oracle.weblogic.kubernetes.assertions.impl.Kubernetes.isPodRestarted;
 import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.awaitility.Awaitility.with;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+
```

Comment:
Please remove extra spaces

Reply:
removed

Reply:
I was trying to get the test to pass on Kind cluster and tried to see if retry works. Didn't want to remove the commented code till I found the solution. Finally passed in kind cluster. So, removed all commented code.